src/common/utils/http.ts

```diff
@@ -0,0 +1,33 @@
+// ==LICENSE-BEGIN==
+// Copyright 2017 European Digital Reading Lab. All rights reserved.
+// Licensed to the Readium Foundation under one or more contributor license agreements.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file exposed on Github (readium) in the project repository.
+// ==LICENSE-END==
+
+import { app } from "electron";
+import { container } from "readium-desktop/main/di";
+import { RootState } from "readium-desktop/main/redux/states";
+import { Store } from "redux";
+import * as request from "request";
+import { promisify } from "util";
+
+export type httpResponse = request.Response;
+
+export function httpGet(url: string, options?: request.CoreOptions): Promise<request.Response> {
+
+    const store = container.get("store") as Store<RootState>;
+    const locale = store.getState().i18n.locale;
+    const requestOptions = Object.assign(
+        {},
+        { url },
+        {
+            headers: {
+                "User-Agent": "readium-desktop",
+                "Accept-Language": `${locale}-${locale.toUpperCase},en-US;q=0.7,en;q=0.5`,
```

Comment:
I think the function should be invoked inside the string template literal, i.e. `${locale.toUpperCase()}` instead of `${locale.toUpperCase}`.

Reply:
I have removed fr-FR

We handle only the first language locale.

It's works on feedbooks